### DISCUSSION TITLE
Question about Picture-in-picture view

### DISCUSSION THREAD

**Discussion by bjspi** [InfinityLoop1308/PipePipe#386] [Q&A]
Hi @InfinityLoop1308 

Thanks for making such a great and awesome enhancement of NewPipe, it's really nice.
One question would be regarding PIP mode:

The PIP (Picture-in-picture) seems a bit weird: First of all it takes a moment to initialize the popup view mode, as if it'd restart the video-view internally, which always causes some sort of delay. 

But what really makes me wonder: How to get back from PIP view to "normal" App-view without immediately going into Fullscreen mode? I mean, sometimes I'm going into PIP mode to use other apps like Whatsapp for chatting - but afterwards I want to go to the regular App view including the comment section etc - but I cannot find a direct transition. Instead, I have to clo video playback from there. It's really cumbersome if switching back and forth.

Any idea or am I missing something?

Thanks!

**Comment by InfinityLoop1308** [comment]
Hi @bjspi , sorry for late response.
I really forget what things behave at that moment, but I just tested in the latest version and the app is able to jump directly back to the video page without entering full screen.
If you still have issues, feel free to reply here.
